Source/Core/VideoCommon/VertexLoaderManager.cpp

```diff
@@ -321,45 +323,102 @@ void LoadCPReg(u32 sub_cmd, u32 value, bool is_preprocess)
   CPState* state = is_preprocess ? &g_preprocess_cp_state : &g_main_cp_state;
   switch (sub_cmd & CP_COMMAND_MASK)
   {
+  case UNKNOWN_00:
+  case UNKNOWN_10:
+  case UNKNOWN_20:
+    if (!(sub_cmd == UNKNOWN_20 && value == 0))
+    {
+      // All titles using libogc or the official SDK issue 0x20 with value=0 on startup
+      DolphinAnalytics::Instance().ReportGameQuirk(GameQuirk::USES_CP_PERF_COMMAND);
+      DEBUG_LOG_FMT(VIDEO, "Unknown CP command possibly relating to perf queries used: {:02x}",
+                    sub_cmd);
+    }
+    break;
+
   case MATINDEX_A:
+    if (sub_cmd != MATINDEX_A)
+    {
+      DolphinAnalytics::Instance().ReportGameQuirk(GameQuirk::USES_MAYBE_INVALID_CP_COMMAND);
+      WARN_LOG_FMT(VIDEO,
+                   "CP MATINDEX_A: an exact value of {:02x} was expected "
+                   "but instead a value of {:02x} was seen",
+                   sub_cmd, MATINDEX_A);
```

Comment:
Shouldn't this be MATINDEX_A and sub_cmd (in this order) instead?
```suggestion
                   MATINDEX_A, sub_cmd);
```

Reply:
Yep.  Probably a leftover from the original message I used.